.drone.yml

```diff
@@ -4,7 +4,6 @@ clone:
 
 matrix:
   PYTHON_VERSION:
-    - 3.5
```

Comment:
I believe the version of python osie is running still uses 3.5 :/

Reply:
wat

Reply:
wow we need to update those containers already

Reply:
put it back and added a comment